verification/curator-service/ui/src/components/bulk-case-form-fields/FileUpload.tsx

```diff
@@ -51,15 +43,26 @@ export default function FileUpload(): JSX.Element {
                         event.currentTarget.files;
                     if (uploadedFiles) {
                         setFieldValue(name, uploadedFiles[0]);
+                        setFileName(
+                            uploadedFiles[0]?.name ? uploadedFiles[0].name : '',
```

Comment:
This can be `uploadedFiles[0]?.name ?? ''`

Reply:
Done, thanks for the catch.